src/noises.jl

```diff
@@ -24,19 +24,147 @@ struct NetworkPauliNoise
     pz::Float64
 end
 
+"Measurement error: chance to report opposite result for coincidence measurement"
+struct MeasurementError 
+    p::Float64
+end
+
+"Thermal relaxation error to be applied per time step. Assuming same time step for all operations, same errors for all qubits"
+struct T1T2Noise
+    t1::Float64  # T1 time constant
+    t2::Float64  # T2 time constant
+end
+
 "A convenience constructor for unbiased Pauli network noise that results in a Bell pair of given fidelity"
 NetworkFidelity(f) = NetworkPauliNoise(f_in_to_pauli(f)...)
 
 function noisify_circuit(n::NetworkPauliNoise, circuit; number_registers)
     initial_noise = [PauliNoiseOp(i, n.px, n.py, n.pz) for i in 1:number_registers]
     return [initial_noise; noisify.((n,), circuit)]
 end
+
 noisify(n::NetworkPauliNoise, b::BellMeasure) = NoisyBellMeasureNoisyReset(b, 0, n.px, n.py, n.pz)
 noisify(n::NetworkPauliNoise, b::NoisyBellMeasureNoisyReset) = NoisyBellMeasureNoisyReset(b.m, b.p, n.px, n.py, n.pz)
 
+function noisify_circuit(error::MeasurementError, circuit; number_registers)
+    return map(op -> noisify(error, op), circuit)
+end
```

Comment:
double check if this is actually needed, because I think there should be a default for it